requirements/development.txt

```diff
@@ -66,6 +64,8 @@ mccabe==0.6.1
     # via pylint
 mysqlclient==2.1.0
     # via apache-superset
+oauthlib==3.2.2
```

Comment:
I don't think you need oauthlib for LDAP

Reply:
Thanks, I checked - this library is really not needed for ldap to work, after its removal, LDAP works correctly.
Removed from PR.
Thanks for the help!